src/Sulu/Bundle/AdminBundle/Resources/js/containers/Sidebar/types.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import type {ComponentType} from 'react';
+
+export type SidebarViewOptions = {};
+
+export type SidebarView = ComponentType<SidebarViewOptions>;
+
+export type Sizes = 'small' | 'medium' | 'large';
+
+export type SidebarConfig = {
+    view?: string,
+    props?: Object,
+    sizes?: Array<string>,
+    defaultSize?: Sizes,
+};
+
+const DEFAULT_SIZE = 'medium';
+const SIZES = ['small', 'medium', 'large'];
+
+export {DEFAULT_SIZE, SIZES};
```

Comment:
I would define these values in the `Sidebar.js` file, because these are values, and not types.